internal/gensupport/resumable_test.go

```diff
@@ -280,7 +280,7 @@ func TestCancelUploadBasic(t *testing.T) {
 	defer func() { backoff = oldBackoff }()
 
 	res, err := rx.Upload(ctx)
-	if err != context.Canceled {
+	if strings.Contains(err.Error(), "context cancelled") {
```

Comment:
Why is this change needed? Can you use errors.Is?

Reply:
Done.